src/menu_system.cpp

```diff
@@ -999,6 +1002,7 @@ void MenuSystem::handleLoginInputSystem(const std::string &character_buffer, con
             // Specified in Config file!  TODO
             std::cout << "m_is_session_authorized" << std::endl;
             m_current_menu = "main";
+            m_starting_menu = "main"; // These should be pulled from config in future!!
```

Comment:
here is where we need to pull in the current / starting menu from the configuration.